### PR TITLE
WIP: Manual Fixes. JuliaCon 2016 Edition

### PR DESCRIPTION
**Pull request by omus**
I've been re-reading the manual to discover any new features I may have missed over the last year. During my reading so far I've found some issues:
1. "Elementary Functions" section seemed very out of place. Mainly the problem is that it breaks up two sections that both talked about rounding
2.  The "Byte Array Literals" section contained a sentence talking about non-standard string literals conventions. I've moved it to a more appropriate area
3. Found a trivial typo in the "Version Number Literals" section

### CODE REVIEW

**Comment by omus** [comment]
I noticed inconsistency in how the manual refers to language. Sometimes it is referred to "Julia" and sometimes "julia". I've updated the manual (and one place in the REPL help) such that when referring to Julia is always "Julia".

Also, the "Elementary Function" commit (d1cf43c) shouldn't be backported as it that is a 0.5 feature.

File: doc/manual/parallel-computing.rst [hunk 1]
```diff
@@ -860,7 +860,7 @@ If ``io`` is not specified, ``host`` and ``port`` are used to connect.
 For example, a cluster manager may launch a single worker per node, and use that to launch
 additional workers. ``count`` with an integer value ``n`` will launch a total of ``n`` workers,
 while a value of ``:auto`` will launch as many workers as cores on that machine.
-``exename`` is the name of the julia executable including the full path.
+``exename`` is the name of the Julia executable including the full path.
```
Comment (tkelman) [inline doc/manual/parallel-computing.rst]: this might be better as quoted lowercase

Comment (omus) [inline doc/manual/parallel-computing.rst]: I was thinking that "Julia" would refer the the language while "\`julia\`" refers to the command-line program. Does that seem sensible?

Comment (StefanKarpinski) [inline doc/manual/parallel-computing.rst]: Yes, exactly. Probably makes sense to also put `julia` in backticks.

Comment (tkelman) [inline doc/manual/parallel-computing.rst]: Agreed. "julia executable" is pretty clearly the latter.

**Comment by kshyatt** [comment]
AV restarted. @tkelman @StefanKarpinski do the latest changes here look ok?

**Comment by omus** [comment]
Feel free to merge this whenever. I have a couple more corrections to add but I can always make another PR.

File: doc/manual/parallel-computing.rst (hunk 1 above)
Comment (tkelman) [inline doc/manual/parallel-computing.rst]: not yet addressed here

**Pull request merged**